src/test/java/io/confluent/idesidecar/restapi/messageviewer/RecordDeserializerTest.java

```diff
@@ -363,7 +363,7 @@ record TestCase(
     ) {
       static TestCase nonRetryable(int statusCode) {
         // Expect only 1 try when the status code is non-retryable
-        return new TestCase(statusCode, 1, 1, null);
+        return new TestCase(statusCode, 3, 1, null);
```

Comment:
I'm curious -- the second parameter is `maxRetries`. Why are we changing this from 1 to 3, when (per the comment) we "expect only 1 try"?

Reply:
Again, this was a change that slipped in, probably while working on testing: https://github.com/confluentinc/ide-sidecar/pull/178/files#r1861315837

Reply:
PS-- the second param is `expectedTries`.

Reply:
Hmm, I think the second parameter is `maxRetries` per [these lines](https://github.com/confluentinc/ide-sidecar/blob/7574242da2435a9aab6e68e1ac91a5be41473f42/src/test/java/io/confluent/idesidecar/restapi/messageviewer/RecordDeserializerTest.java#L358-L363) just above this change:
```
    record TestCase(
        int statusCode,
        int maxRetries,
        int expectedTries,
        Boolean isKey
    ) {
      ...
```